src/docs/forms/components/RadioGroup/examples/06-simpleValueProperty.js

```diff
@@ -0,0 +1,66 @@
+import React from 'react'
+import Form, {
+    RadioGroup
+} from 'doj-react-adminlte/Form';
+
+export default class simpleValueProperty extends React.Component {
+    constructor(props) {
+        super(props);
+        this.state = {
+            food1: null,
+            food2: null
+        };
+    }
+
+    handleChange = (field, value) => {
+        this.setState({
+            [field]: value
+        });
+    };
+
+    render() {
+        const {food1, food2} = this.state;
+        const foodOptions = [
+            {label: "Spaghetti", value: "spaghetti"},
+            {label: "Fried Chicken", value: "fried_chicken"},
+            {label: "Burger Steak", value: "burger_steak"},
+            {label: "Hamburger", value: "hamburger"}
+        ];
+        const jsonFood = JSON.stringify(food2,null,2);
```

Comment:
spaces after commas

Reply:
make this inline by transferring this directly in return statement. see comment below.